spec/CkmeansNodeModernSpec.js

```diff
@@ -0,0 +1,18 @@
+/* globals describe, it, expect */
+const nativeCkMeans = require('../src/ckmeans-native');
+const jsCkMeans = require('../src/ckmeans');
+describe('Ckmeans', function () {
+  it('Test they return the same', function () {
+    for (let i = 0; i < 30; i++) {
+      const length = parseInt(Math.random() * 10 + 300);
+      const data = [];
+      for (let j = 0; j < length; j++) {
+        data[j] = parseInt(Math.random() * 100);
```

Comment:
Using random values in tests usually isn't ideal since results may not be reproducible across runs. Any way we can make this test deterministic?